layouts/partials/footer.html

```diff
@@ -18,7 +18,7 @@
       </div>
       <div class="td-footer__copyright-etc col-12 col-sm-4 text-center py-2 order-sm-2">
         {{ with .Site.Params.copyright -}}
-          <span>&copy; {{ now.Year }} {{ . }} {{ T "footer_all_rights_reserved" }}</span>
+          <span>&copy; {{ with .year -}}{{- . -}}{{- else -}}{{- now.Year -}}{{- end }} {{ with .title -}}{{- . -}}{{- end }} {{ T "footer_all_rights_reserved" }}</span>
         {{- end }}
```

Comment:
Could I suggest defaulting on the  site copyright config?

```suggestion
          <span>&copy; {{ with .year -}}{{- . -}}{{- else -}}{{- now.Year -}}{{- end }} {{ with .title -}}{{- . -}}{{- end }} {{ T "footer_all_rights_reserved" }}</span>
        {{- else }}
          {{ with .Site.Copyright }}<span>{{ . | safeHTML }}</span>{{ end }}
        {{- end }}
```

Reply:
> Could I suggest defaulting on the site copyright config?

Good idea. Implemented as requested.